packages/server/src/api_auth/index.ts

```diff
@@ -249,8 +249,9 @@ router.post('/signup', apiRequestAuthSignupValidator, async (req, res) => {
  */
 router.post('/magic', apiRequestAuthMagicValidator, async (req, res) => {
     try {
-        const parsedBody = await emailObjectValidator.parseAsync(req.body);
-        const email = parsedBody.email;
+        const reqClientData: IRequestAuthMagic = res.locals.reqClientData;
+        const email: string = reqClientData.body.email;
+        console.log(req.body.email);
```

Comment:
Please make sure to remove any console before submitting PR.

Reply:
sure